src/styles/actionable_row.scss

```
@@ -7,12 +7,28 @@
   border-radius: var(--XS, 8px);
   background: var(--color-base-0);
   box-shadow: 0px 0px 0px 1px var(--color-base-300);
+  container-type: inline-size;
+
+  /* Stack vertically when container is too narrow */
+  @container (max-width: 400px) {
```

```suggestion
  @container (max-width: 30rem) {
```

Let's stack a bit larger so the text doesn't go over two lines (30 rem === 480px) and also prefer to use rem here rather than px values to scale with the user's text size (if they change a11y settings).